ros2component/test/test_cli_load.py

```diff
@@ -0,0 +1,217 @@
+        output_format='{line}', additional_env=additional_env,
+        name='container_' + rmw_implementation)
+    return LaunchDescription([
+        # Always restart daemon to isolate tests.
+        ExecuteProcess(
+            cmd=['ros2', 'daemon', 'stop'],
+            name='daemon-stop',
+            on_exit=[
+                ExecuteProcess(
+                    cmd=['ros2', 'daemon', 'start'],
+                    name='daemon-start',
+                    on_exit=[
+                        component_node,
+                        OpaqueFunction(function=lambda context: ready_fn())
+                    ],
+                    additional_env=additional_env
+                )
+            ]
+        ),
+    ]), locals()
+
+
+class TestROS2ComponentLoadCLI(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(
+        cls,
+        launch_service,
+        proc_info,
+        proc_output,
+        rmw_implementation,
+        component_node
+    ):
+        @contextlib.contextmanager
+        def launch_component_command(self, arguments):
+            component_command_action = ExecuteProcess(
+                cmd=['ros2', 'component', *arguments],
+                additional_env={
+                    'RMW_IMPLEMENTATION': rmw_implementation,
+                    'PYTHONUNBUFFERED': '1'
+                },
+                name='ros2component-cli',
+                output='screen'
+            )
+            with launch_testing.tools.launch_process(
+                launch_service, component_command_action, proc_info, proc_output,
+                output_filter=launch_testing_ros.tools.basic_output_filter(
+                    # ignore ros2cli daemon nodes
+                    filtered_patterns=['.*ros2cli.*'],
+                    filtered_rmw_implementation=rmw_implementation
+                )
+            ) as component_command:
+                yield component_command
+        cls.launch_component_command = launch_component_command
+        cls.component_node = launch_testing.tools.ProcessProxy(
+            component_node, proc_info, proc_output,
+            output_filter=launch_testing_ros.tools.basic_output_filter(
+                    # ignore ros2cli daemon nodes
+                    filtered_patterns=['.*ros2cli.*'],
+                    filtered_rmw_implementation=rmw_implementation
+                )
+        )
+
+    @launch_testing.markers.retry_on_failure(times=2)
+    def test_load_verb(self):
+        with self.launch_component_command(
+                arguments=[
+                    'load', '/ComponentManager',
+                    'ros2component_test_fixtures',
+                    'ros2component_test_fixtures::Talker']) as load_command:
+            assert load_command.wait_for_shutdown(timeout=20)
+            assert self.component_node.wait_for_output(functools.partial(
+                launch_testing.tools.expect_output, expected_lines=[
+                    "[INFO] [talker]: Publishing: 'Hello World'"
```

Comment:
@BMarchi consider asserting container output instead of talker output.

Reply:
This comes from the container instead of the talker, everything prints out in the process where the container is run